src/main/java/org/prebid/server/bidder/nextmillennium/NextMillenniumBidder.java

```diff
@@ -51,12 +52,46 @@ public NextMillenniumBidder(String endpointUrl, JacksonMapper mapper) {
     }
 
     private static BidRequest updateBidRequest(BidRequest bidRequest, ExtImpNextMillennium ext) {
+        ExtRequest extRequest = ExtRequest.of(ExtRequestPrebid.builder()
+                .storedrequest(ExtStoredRequest.of(resolveStoredRequestId(bidRequest, ext)))
+                .build());
+        List<Imp> imps = bidRequest.getImp()
+                .stream()
+                .map(imp -> Imp.builder()
+                        .id(imp.getId())
```

Comment:
You can use `toBuilder()` function to create new object and replace only required values

Reply:
@SerhiiNahornyi thank you very much! Didn't work with lombok too much